docs/reference/mapping/params/boost.asciidoc

```diff
@@ -1,8 +1,8 @@
-[[index-boost]]
+[[mapping-boost]]
```

Comment:
Since you are removing an existing link, I think you need to add an entry to redirects.asciidoc?